v1/ros-noetic.yaml

```diff
@@ -0,0 +1,82 @@
+description: A development and testing environment for ROS Noetic.
+version: latest
```

Comment:
I know many other Blueprints use `latest` here, but I'd like to get away from that convention and put in a meaningful number such as `0.1` or `1.0`, etc.

Reply:
Hi @townsend2010, just to be sure, this versionning is that of this yaml file or is it the version of the generated instance? In the former case I would go with `0.1` and increase it with subsequent changes whereas in the later case I'd mark it as `noetic` and live it as is.

Reply:
Hi @artivis, the intent of the `version` field is for the versioning of this yaml file.

Reply:
Thanks. I've changed it to 0.1.